crates/bevy_core_pipeline/src/tonemapping/tonemapping.wgsl

```diff
@@ -10,5 +10,15 @@ var hdr_sampler: sampler;
 fn fragment(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
     let hdr_color = textureSample(hdr_texture, hdr_sampler, in.uv);
 
-    return vec4<f32>(reinhard_luminance(hdr_color.rgb), hdr_color.a);
+    var output_rgb = vec3<f32>(reinhard_luminance(hdr_color.rgb));
```

Comment:
Nit: if Reinhard luminance returns a vec3 then wrapping it in one is unnecessary.

Reply:
The intermediate variable is needed to work with the ifdef, and reassemble a vec4 with the alpha value. Right?

Reply:
I wasn’t suggesting to remove the variable. I was just saying that if the function returns a vec3 then we don’t need to do vec3(func()), just func().

Reply:
Oh, duh! Thanks for clarifying. 🙂